daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -568,6 +570,27 @@ createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not
         doesFileExist (cdaDir </> "ui" </> "build" </> "index.html") >>=
           assertBool "ui/build/index.html has been produced"
 
+        -- Now test that the messaging feature works by applying the necessary
+        -- changes and testing in the same way as above.
+        messagingPatch <- locateRunfiles (mainWorkspace </> "templates" </> "messaging-patch")
+        withCurrentDirectory cdaDir $ do
+          callCommandSilent $ "patch -s -p2 < " ++ messagingPatch
+          forM_ ["MessageEdit", "MessageList"] $ \messageComponent ->
+            doesFileExist ("ui" </> "src" </> "components" </> messageComponent <.> "tsx") >>=
+              assertBool ("New " <> messageComponent <> " component was not created by patch")
+              . not
+          callCommandSilent "daml build"
+          setupYarnEnv tmpDir (Workspaces ["create-daml-app/daml.js"]) [DamlTypes]
+          callCommandSilent "daml codegen js -o daml.js .daml/dist/create-daml-app-0.1.0.dar"
+        doesFileExist (cdaDir </> "ui" </> "build" </> "index.html") >>=
+          assertBool "ui/build/index.html does not yet exist" . not
```

Comment:
Why would this file not exist anymore? I suggest we remove the whole whole `ui/build` directory once the first half of the test is done.